notebooks/bimodal_dist_plot.ipynb

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    plt.rc("text", usetex=True)
</pre><p>This is already present in latexify but not needed here. We can remove it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/pyprobml-fork/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>